### ISSUE TITLE
csysdig broken on rxvt-unicode

### ISSUE THREAD

**Issue by ghost** [draios/sysdig#689]
OS: Arch Linux
Kernel: 4.8.10-1-ARCH

I use rxvt-unicode as my terminal emulator, and after installing the `sysdig` and `linux-headers` packages, I went to try out csysdig for the first time. When I lauched it, the screen wouldn't stop flashing, and many things were broken. Pressing F1 to go into help would only open for a split second, and when I clicked on the Help button, it only stayed open for about 1~2 seconds (until the main process list refreshed.) Thinking that this is some weird ncurses issue, I installed xterm and tried it there. It worked just fine, but I really am not switching to xterm anytime soon. I also thought that it could have been an issue with my fonts. I use a bitmap font, and I have seen problems with it before. That was not the case, though. I hope this gets fixed soon, because this looks like amazing software. 

TL;DR
Something with ncurses makes csysdig become unusable in rxvt-unicode.

**Comment by hugecannon** [comment]
I seem to be experiencing this now, too. 

Any ideas on what may cause this?

**Comment by erikarvstedt** [comment]
Same here, `rxvt-unicode (urxvt) v9.22 - released: 2016-01-23`
